spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -182,7 +182,7 @@ const ConfigSteps = () => {
 				} catch (e) {
 					setLoaderForLogin(false);
 					showError(modifyError(e as AxiosError, {}, { onClearGitHubToken: clearGitHubToken, onRelogin: reLogin }));
-					reportError(e, {
```

Comment:
We have to call the `new Error(xxx)` here instead of inside the `reportError` because the stacktrace will only be caught during the calling of `new Error()`

Reply:
And wrap it with a `new Error` will help Sentry group the error into different issues.